### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by Emerkof**
(Done)1 - File serverless.yml contains configuration for importProductsFile function
(Done)3 - The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
(Done)4 - Frontend application is integrated with importProductsFile lambda
(Done)5 - The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda

(Done)+1 - async/await is used in lambda functions
(Done)+1 - importProductsFile lambda is covered by unit tests
(Done)+1 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder

total: 8

Lambda import link: https://vapcj1zvui.execute-api.eu-west-1.amazonaws.com/dev/import
FE PR: https://github.com/Emerkof/nodejs-aws-fe/pull/4/
CloudFront: https://d158ao7f9hlam3.cloudfront.net/

### CODE REVIEW

**Comment by mmozhorov** [comment]
I'll review it )
@Rider98113

**Comment by BohdanRSchool** [comment]
Waiting for the PR

**Comment by Emerkof** [comment]
@Rider98113 @BohdanRSchool 

Ready for review :)

**Comment by Emerkof** [comment]
@Major98113 @BohdanRSchool 

Ready for review :)

File: lib/aws/s3.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  copyObject(params = {}) {
+    return this._s3.copyObject(params).promise();
+  }
+
+  deleteObject(params = {}) {
+    return this._s3.deleteObject(params).promise();
+  }
+}
+
+export default new AwsS3();
```
Comment (sleepwalky) [inline lib/aws/s3.js:25]: what is the motivation to have your own wrapper around aws-sdk?

File: import-service/handlers/import-products-file.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { handleRequestResponseDecorator } from '../../lib/response';
+import { BadRequestError } from '../../lib/error';
+
+import ImportService from '../services/import';
+
+export const getImportProductSignedUrl = handleRequestResponseDecorator(
+  async function(event) {
+    const { queryStringParameters: { name } = {} } = event;
```
Comment (ezhukauskas) [inline import-service/handlers/import-products-file.js:8]: If I do not pass any query parameters, then queryStringParameters=null by spec and this line will fail.

Just try https://vapcj1zvui.execute-api.eu-west-1.amazonaws.com/dev/import and you'll get 500:
{"error":{"message":"Something went wrong, please, try again later"}}

Try non-empty query (but not name): https://vapcj1zvui.execute-api.eu-west-1.amazonaws.com/dev/import?unknown=2 and you'll get correct 400

File: lib/aws/s3.js (hunk 1 above)
Comment (Emerkof) [reply lib/aws/s3.js:25]: @sleepwalky I always try to encapsulate work with third-party in single file. If I suddenly decide to replace a third-party dependency with another module, I will only have to modify the code in 1 place, adjusting methods' implementation, whereas lib's interface will remain intact

**Pull request merged**